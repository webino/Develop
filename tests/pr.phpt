<?php
/**
 * Webino™ (http://webino.sk)
 *
 * @outputmatchfile pr.expected
 *
 * @link        https://github.com/webino/develop
 * @copyright   Copyright (c) 2019 Webino, s.r.o. (http://webino.sk)
 * @author      Peter Bačinský <peter@bacinsky.sk>
 * @license     BSD-3-Clause
 */

Tester\Environment::setup();


$a = ['A', 'B'];


echo pr($a);
