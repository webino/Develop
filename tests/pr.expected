<pre class="tracy-dump"><span class="tracy-toggle"><span class="tracy-dump-array">array</span> (2)</span>
<div><span class="tracy-dump-indent">   </span><span class="tracy-dump-key">0</span> => <span class="tracy-dump-string">"A"</span>
<span class="tracy-dump-indent">   </span><span class="tracy-dump-key">1</span> => <span class="tracy-dump-string">"B"</span>
</div></pre>
